hardware_tests/test_polarization_controller_thorlabs_mpc320.py

```diff
@@ -75,6 +75,25 @@ def test_move_absolute(device: PolarizationControllerThorlabsMPC320) -> None:
     # device.move_absolute(ChanIdent.CHANNEL_3, 0 * ureg.degree)
 
 
+def test_jog(device: PolarizationControllerThorlabsMPC320) -> None:
+
+    jog_step_1 = 50
+    jog_count_1 = 5
+
+    device.set_params(jog_step_1=jog_step_1, jog_step_2=25, jog_step_3=25)
```

Comment:
You can just set `jog_step_1`, you don't need to touch the other two.
```suggestion
    device.set_params(jog_step_1=jog_step_1)
```